x/pool-incentives/keeper/keeper_test.go

```diff
@@ -96,62 +96,103 @@ func (suite *KeeperTestSuite) TestCreateConcentratePoolGauges() {
 	}
 }
 
-func (suite *KeeperTestSuite) TestCreatePoolGauges() {
+func (suite *KeeperTestSuite) TestCreateLockablePoolGauges() {
 	tests := []struct {
-		name        string
-		poolId      uint64
-		poolType    poolmanagertypes.PoolType
-		expectedErr bool
+		name                   string
+		poolId                 uint64
+		expectedGaugeDurations []time.Duration
+		expectedGaugeIds       []uint64
+		expectedErr            bool
 	}{
 		{
-			name:        "Concentrated Liquidity Pool",
-			poolId:      1,
-			poolType:    poolmanagertypes.Concentrated,
-			expectedErr: false,
+			name:                   "Create Gauge with valid PoolId",
+			poolId:                 uint64(1),
+			expectedGaugeDurations: []time.Duration{time.Hour, time.Hour * 3, time.Hour * 7},
```

Comment:
Let's not hard-code these, please. Theses should be equal to lockable durations that can be retrieved by a relevant keeper method